public/locales/bg/common.json

```diff
@@ -9,7 +9,7 @@
     },
     "campaigns": {
       "index": "Кампании",
-      "create": "Предложи кампания",
+      "create": "Създай кампания",
```

Comment:
Marketing and other teams requested this to be renamed to Suggest campaign to avoid awkward moments that the campaign will be created straight away without reviewing.

Anyway I'm fine with it